packages/serverless-contracts/src/contracts/apiGateway/features/axiosRequest.ts

```diff
@@ -0,0 +1,23 @@
+import { AxiosInstance, AxiosResponse } from 'axios';
+
+import { ApiGatewayContract } from '../apiGatewayContract';
+import { OutputType, RequestArguments } from '../types';
+import { getRequestParameters } from './requestParameters';
+
+export const getAxiosRequest = async <Contract extends ApiGatewayContract>(
+  contract: Contract,
+  axiosClient: AxiosInstance,
+  requestArguments: RequestArguments<Contract>,
+): Promise<AxiosResponse<OutputType<Contract>>> => {
+  const { path, method, queryStringParameters, body, headers } =
+    // @ts-ignore: Type instantiation is excessively deep and possibly infinite.
```

Comment:
Hmm this probably means that our types are not efficient. We can look at this together